src/ray/common/cgroup/base_cgroup_setup.h

```diff
@@ -42,17 +42,10 @@ class BaseCgroupSetup {
   BaseCgroupSetup &operator=(const BaseCgroupSetup &) = delete;
 
   // Add system process into system cgroup.
-  virtual ScopedCgroupHandler AddSystemProcess(pid_t pid) = 0;
+  virtual Status AddInternalProcess(pid_t pid) = 0;
 
   // Apply cgroup context, which adds the process id into the corresponding cgroup.
   virtual ScopedCgroupHandler ApplyCgroupContext(const AppProcCgroupMetadata &ctx) = 0;
-
- protected:
-  // Remove the given system process [pid] from system cgroup.
-  virtual void CleanupSystemProcess(pid_t pid) = 0;
```

Comment:
We don't need a util function to cleanup internal processes, because all of them are placed at the same cgroup folder, and gets removed from the PID file after they complete.

The only thing we need to do, is to delete the delete the internal sub-cgroup after raylet exits, using https://github.com/ray-project/ray/blob/master/src/ray/util/process_cleaner.h